.github/workflows/token-dashboard.yml

```
@@ -0,0 +1,87 @@
+name: Token Dashboard dApp
+
+#TODO: extend the conditions once workflow gets tested together with other workflows
+on:
+  push:
+    branches:
+      # TODO: Run on master after we're fully migrated from Circle CI
+      # - master
+      - "rfc-18/**"
+    paths:
+      - "solidity/dashboard/**"
+      - ".github/workflows/token-dashboard.yml"
+  pull_request:
+    branches:
+      # TODO: Run on all branches after we're fully migrated from Circle CI
+      - "rfc-18/**"
+    paths:
+      - "solidity/dashboard/**"
+      - ".github/workflows/token-dashboard.yml"
+
+jobs:
+  build-publish-token-dashboard-dapp:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./solidity/dashboard
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "12.x"
```

Can we use the matrix strategy solution proposed in https://github.com/keep-network/keep-core/pull/2326/files#diff-a13b95356a0820b9ce4307a209d93d7f0da6e66acabb3bbb4a4cc3e172662931R28-R31?
```suggestion
    - name: Use Node.js ${{ matrix.node-version }}
      uses: actions/setup-node@v2
      with:
        node-version: ${{ matrix.node-version }}
```

---

See deeb23bdf16c3ba9be56a962e419eec04b7d5947